notebooks/MAST/tess_cutout_rotation_curve/tess_cutout_rotation_curve.ipynb

```diff
@@ -0,0 +1,810 @@
+{
```

Comment:
<p>First sentence: The word "to" is repeated at the beginning.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>